### ISSUE TITLE
Jetpack references nonexistent file (modules/sharing/sharing-service.php)

### ISSUE THREAD

**Issue by paulschreiber** [Automattic/jetpack#4199]
In `class.core-rest-api-endpoints.php`, Jetpack references a nonexistent file (`modules/sharing/sharing-service.php`):

```
$ grep -r 'modules/sharing/sharing-service.php' .
./_inc/lib/class.core-rest-api-endpoints.php:		if ( ! class_exists( 'Sharing_Service' ) && ! @include( JETPACK__PLUGIN_DIR . 'modules/sharing/sharing-service.php' ) ) {
./_inc/lib/class.core-rest-api-endpoints.php:		if ( ! class_exists( 'Sharing_Service' ) && ! @include( JETPACK__PLUGIN_DIR . 'modules/sharing/sharing-service.php' ) ) {
./_inc/lib/class.core-rest-api-endpoints.php:		if ( ! class_exists( 'Sharing_Service' ) && ! @include( JETPACK__PLUGIN_DIR . 'modules/sharing/sharing-service.php' ) ) {
./_inc/lib/class.core-rest-api-endpoints.php:				if ( ! class_exists( 'Sharing_Service' ) && ! @include( JETPACK__PLUGIN_DIR . 'modules/sharing/sharing-service.php' ) ) {

$ ls modules/sharing/sharing-service.php
ls: modules/sharing/sharing-service.php: No such file or directory
```

This should be `modules/sharedaddy/sharing-service.php`.

**Comment by paulschreiber** [comment]
This was introduced in ee6a46c9 by @eliorivero.

**Comment by eliorivero** [comment]
Thanks for the heads up Paul. It's now fixed in #4203

**Comment by paulschreiber** [comment]
@eliorivero I had already fixed it — see my above commit (eed0cf3).
